tests/e2e/pact/models/cases/process-case.ts

```diff
@@ -0,0 +1,17 @@
+import {Bundle} from "../fhir/fhir-resources"
```

Comment:
These classes share some common fields and the constructors are all quite similar, could they be cleaned up through inheritance?. If this ticket is needed urgently might be worth raising another ticket to do it?

Reply:
Yep a refactor here would be ideal in another ticket